docs/src/Model_Reference/Resources/fusion.md

```diff
@@ -0,0 +1,289 @@
+The default of no restriction is entered with `-1`.
+Note that if the plant has a dwell time $t_\mathrm{dw} > 0$ and a maximum up time, the theoretical maximum gross power production is already less than one full-power year per year.
+For example, if there is a 0.5 hour dwell time and a max up time of 2 hours, then the plant can operate at peak power for 1.5 hours out of every 2, which yields a theoretical maximum throughput of 0.75 FPY per year.
+In this case a `max_fpy_per_year` of 0.75 or more would have no effect.
+
+#### Modification to parasitic power during maintenance
+
+The `parasitic_passive_maintenance_remaining` parameter is only used for plants also using the scheduled maintenance module (`maint = 1`).
+The value must be between $0$ and $1$.
+The default is $0$; in this case there is no parasitic power during scheduled maintenance periods.
+With a value of $1$ the parasitic power does not decrease during maintenance.
+With a value of $0.5$ it is reduced by half, and so on.
+
+## Outputs produced
+
+Use of the fusion module leads to two new outputs files, and changes to existing output files.
+
+- `fusion_pulse_starts`. This file is similar to `start/commit/shutdown.csv` for unit committment.
+There is a column for each fusion resource component and a row for each timestep.
+Values are number of plant units (of a given resource component) starting a fusion pulse in each timestep.
+
+- `fusion_parasitic_power`. This file is similar to `charge.csv`.
+There is a column for each fusion resource component and a row for each timestep.
+Values are the parasitic energy in MWh drawn by each resource component in each timestep.
+This is the sum of the passive parasitic power, the active parasitic power, and the pulse start energy.
+Values are always nonnegative. Positive values represent power flowing from the grid to the plant.
+
+- `fusion_net_capacity_factor`. This is the ratio of the net power production (gross - parasitic) to the annual net capacity of the plant. The numerator here is always smaller than that of the standard capacity factor and could even be negative if the parasitic power is very large and the plant is not used often. The denominator is also smaller than that in the standard capacity factor because it accounts for parasitic power and dwell time.
+
+### Notes on other outputs
+
+- `capacity_factor`: This standard output file reports the _gross_ capacity factor for the plant:
+The ratio of the gross power produced to (the gross capacity * one year). Compare with the `fusion_net_capacity_factor` above.
+
+- `charge`: fusion parasitic power is reported in this file.
+
+## Mathematical formulation of fusion module
+
+### Pulse start and pulse status
+There are two new variables, representing the number of pulse starts ($\chi^{pulse}_{y,z,t}$) and pulses underway ($\nu^{pulse}_{y,z,t}$).
+These are similar to the conventional unit commitment start $\chi$ and commit status $\nu$.
+
+`TODO: tidy up math presentation?`
```

Comment:
Is this still a work in progress? The math part looks great.

Reply:
done.